[BUG] issues with latex_example_full

`latex_example_base` works fine, but `latex_example_full` has issues:
```
bk22rc ❯ python latex_extension_full.py  
Working directory: /home/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label
Using /home/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label/tsconfig.json
Compiling TypeScript (2 files)
latex_label/index.ts:3:31 - error TS2307: Cannot find module '@bokehjs/base'.

3 import {register_models} from "@bokehjs/base"
                                ~~~~~~~~~~~~~~~
latex_label/latex_label.ts:1:32 - error TS2307: Cannot find module '@bokehjs/models/annotations/label'.

1 import {Label, LabelView} from "@bokehjs/models/annotations/label"
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
latex_label/latex_label.ts:14:24 - error TS2339: Property 'angle_units' does not exist on type 'LatexLabel'.

14     switch (this.model.angle_units) {
                          ~~~~~~~~~~~
latex_label/latex_label.ts:16:33 - error TS2339: Property 'angle' does not exist on type 'LatexLabel'.

16         angle = -1 * this.model.angle
                                   ~~~~~
latex_label/latex_label.ts:20:33 - error TS2339: Property 'angle' does not exist on type 'LatexLabel'.

20         angle = -1 * this.model.angle * Math.PI/180.0
                                   ~~~~~
latex_label/latex_label.ts:27:24 - error TS2339: Property 'panel' does not exist on type 'LatexLabelView'.

27     const panel = this.panel || this.plot_view.frame
                          ~~~~~
latex_label/latex_label.ts:27:38 - error TS2339: Property 'plot_view' does not exist on type 'LatexLabelView'.

27     const panel = this.panel || this.plot_view.frame
                                        ~~~~~~~~~
latex_label/latex_label.ts:29:12 - error TS2339: Property 'x' does not exist on type 'LatexLabel'.

29     const {x, y} = this.model
              ~
latex_label/latex_label.ts:29:15 - error TS2339: Property 'y' does not exist on type 'LatexLabel'.

29     const {x, y} = this.model
                 ~
latex_label/latex_label.ts:30:25 - error TS2339: Property 'x_units' does not exist on type 'LatexLabel'.

30     let sx = this.model.x_units == "data" ? this.coordinates.x_scale.compute(x) : panel.xview.compute(x)
                           ~~~~~~~
latex_label/latex_label.ts:30:50 - error TS2339: Property 'coordinates' does not exist on type 'LatexLabelView'.

30     let sx = this.model.x_units == "data" ? this.coordinates.x_scale.compute(x) : panel.xview.compute(x)
                                                    ~~~~~~~~~~~
latex_label/latex_label.ts:31:25 - error TS2339: Property 'y_units' does not exist on type 'LatexLabel'.

31     let sy = this.model.y_units == "data" ? this.coordinates.y_scale.compute(y) : panel.yview.compute(y)
                           ~~~~~~~
latex_label/latex_label.ts:31:50 - error TS2339: Property 'coordinates' does not exist on type 'LatexLabelView'.

31     let sy = this.model.y_units == "data" ? this.coordinates.y_scale.compute(y) : panel.yview.compute(y)
                                                    ~~~~~~~~~~~
latex_label/latex_label.ts:33:22 - error TS2339: Property 'x_offset' does not exist on type 'LatexLabel'.

33     sx += this.model.x_offset
                        ~~~~~~~~
latex_label/latex_label.ts:34:22 - error TS2339: Property 'y_offset' does not exist on type 'LatexLabel'.

34     sy -= this.model.y_offset
                        ~~~~~~~~
latex_label/latex_label.ts:36:10 - error TS2339: Property '_css_text' does not exist on type 'LatexLabelView'.

36     this._css_text(this.layer.ctx, "", sx, sy, angle)
            ~~~~~~~~~
latex_label/latex_label.ts:36:25 - error TS2339: Property 'layer' does not exist on type 'LatexLabelView'.

36     this._css_text(this.layer.ctx, "", sx, sy, angle)
                           ~~~~~
latex_label/latex_label.ts:37:29 - error TS2339: Property 'text' does not exist on type 'LatexLabel'.

37     katex.render(this.model.text, this.el!, {displayMode: true})
                               ~~~~
latex_label/latex_label.ts:37:40 - error TS2339: Property 'el' does not exist on type 'LatexLabelView'.

37     katex.render(this.model.text, this.el!, {displayMode: true})
                                          ~~
latex_label/latex_label.ts:45:20 - error TS2339: Property 'default_view' does not exist on type 'LatexLabel'.

45     this.prototype.default_view = LatexLabelView
                      ~~~~~~~~~~~~
latex_label/latex_label.ts:47:10 - error TS2339: Property 'override' does not exist on type 'typeof LatexLabel'.

47     this.override({
            ~~~~~~~~

Linking modules
Output written to /home/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label/dist
All done.

```

![image](https://user-images.githubusercontent.com/1078448/90593060-e5cfc200-e19b-11ea-81aa-19074bcb75e3.png)

noting that the error message has shortened on latest `branch-2.3`
```
(dev) ❯ bokeh build --rebuild
Working directory: /Users/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label
Using different version of bokeh, rebuilding from scratch.
Running npm install.
added 198 packages from 265 contributors and updated 115 packages in 6.13s
Using /Users/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label/tsconfig.json
Compiling TypeScript (2 files)
latex_label.ts:42:23 - error TS2503: Cannot find namespace 'p'.

42   export type Attrs = p.AttrsOf<Props>
                         ~
latex_label.ts:42:23 - error TS4081: Exported type alias 'Attrs' has or is using private name 'p'.

42   export type Attrs = p.AttrsOf<Props>
                         ~

Linking modules
Output written to /Users/bryan/work/bokeh/examples/custom/latex_extension_full/latex_label/dist
All done.
```

--- a/examples/custom/latex_extension_base/latex_extension_base.py
+++ b/examples/custom/latex_extension_base/latex_extension_base.py
@@ -8,9 +8,8 @@ from bokeh.palettes import Spectral4
 from bokeh.plotting import figure, output_file, show
 from latex_label import LatexLabel
 
-build("latex_label")
-
-
+if not build("latex_label"):
+    raise RuntimeError("unable to build latex_label extension")
 
 output_file('latex_extension.html')
 

--- a/examples/custom/latex_extension_base/latex_label/latex_label.ts
+++ b/examples/custom/latex_extension_base/latex_label/latex_label.ts
@@ -1,4 +1,5 @@
 import {Label, LabelView} from "@bokehjs/models/annotations/label"
+import * as p from "@bokehjs/core/properties"
 
 declare namespace katex {
   function render(expression: string, element: HTMLElement, options: {displayMode?: boolean}): void

--- a/examples/custom/latex_extension_full/latex_extension_full.py
+++ b/examples/custom/latex_extension_full/latex_extension_full.py
@@ -8,9 +8,8 @@ from bokeh.palettes import Spectral4
 from bokeh.plotting import figure, output_file, show
 from latex_label import LatexLabel
 
-build("latex_label")
-
-
+if not build("latex_label"):
+    raise RuntimeError("unable to build latex_label extension")
 
 output_file('latex_extension.html')
 

--- a/examples/custom/latex_extension_full/latex_label/latex_label.ts
+++ b/examples/custom/latex_extension_full/latex_label/latex_label.ts
@@ -1,9 +1,7 @@
 import {Label, LabelView} from "@bokehjs/models/annotations/label"
+import * as p from "@bokehjs/core/properties"
 
 import * as katex from "katex"
-
-//"https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.10.0/katex.min.js"
-//"https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.10.0/katex.min.css"
 
 export class LatexLabelView extends LabelView {
   model: LatexLabel

